ios/FluentUI/People Picker/PeoplePicker.swift

```diff
@@ -92,6 +92,11 @@ open class PeoplePicker: BadgeField {
      */
     @objc open var allowsPickedPersonasToAppearAsSuggested: Bool = true
 
+	/**
+	 Set `hidePersonaListViewWhenNoSuggestedPersonas` to true to hide the personaListView when no suggested personas is available, i.e. personaListView is empty.
```

Comment:
nit on the wording: Set `hidePersonaListViewWhenNoSuggestedPersonas` to true to hide the personaListView when no suggested personas **are** available

Reply:
Done. Thanks for catching it :)